guides/delivery/delivery-lead-guide-pen-tests.md

```diff
@@ -0,0 +1,13 @@
+# Delivery leads guide for arranging pen tests
+
+When a penetration test is arranged by a client on a site which is hosted by
```

Comment:
This sentence feels like a run-on sentence. Maybe add a couple of commas: 

`When a penetration test is arranged by a client on a site which is hosted by dxw digital, the delivery leads must find out the scope of the test and raise a ticket in Zendesk, as the Ops team will need to notify AWS as part of our contract with them.`

Reply:
Yes that looks better

Reply:
@olivia-work Given that this is your PR, would you mind one of us making the above changes, or do you want to make them yourself?